autogluon/utils/tabular/ml/models/abstract/abstract_model.py

```diff
@@ -228,19 +233,45 @@ def compute_feature_importance(self, X, y, features_to_use=None):
         X.reset_index(drop=True, inplace=True)
         y.reset_index(drop=True, inplace=True)
 
-        X = self.preprocess(X)
+        if preprocess:
+            X = self.preprocess(X)
 
         if not features_to_use:
             features = X.columns.values
         else:
             features = features_to_use
-        feature_count = len(features)
 
-        model_score_base = self.score(X=X, y=y)
+        feature_importance_quick_dict = self.get_model_feature_importance()
+        # TODO: Also consider banning features with close to 0 importance
```

Comment:
I don't think we need to ban features with ~0 importance, as long as: A) we are sorting them based on importance in the presented output, B) we are using an output format (like pandas) that will automatically truncate lengthy outputs when printing them.